src/Middleware/RetryMiddleware.php

```diff
@@ -128,11 +128,11 @@ private function retry(Call $call, array $options, string $status)
         }
 
         $delayMs = min($delayMs * $delayMult, $maxDelayMs);
-        $timeoutMs = min(
+        $timeoutMs = (int) floor(min(
```

Comment:
Why not just cast the result of `min` to an int? Why do we need the `floor` here? If you cast a float to an int, it already rounds down to the nearest int.

Reply:
Just that it was the middle of the night & I'd forgotten that casting truncates rather than rounding :) Thanks for the reminder. I will amend.